notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<p>    For each parent review comment we append following,</p><p>    1. Small message 'Reply via ReviewNB' with hyperlink to the PR page.</p><p>       So people can quickly participate in the discussion (and see the context cell while they reply).</p><p>    2. Hidden &lt;div&gt; tag that carries custom data to identify context cell &amp; current status (OPEN/RESOLVED) of the thread.</p>


 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>